### PR TITLE
Changes related to NA-473 and NA-475

### PR DESCRIPTION
**Pull request by nikhildoifode**
Changes:
- Changes UE -> TERMINAL and UE APPLICATION -> TERMINAL APPLICATION in front-end along with suggested element name
- All open panes are closed when scenario are terminated or sandbox is destroyed.

Test:
- All UT's and Cypress tests passed
- Manually checked all changes

### CODE REVIEW

File: js-apps/meep-frontend/src/js/meep-constants.js [hunk 1]
```diff
@@ -215,9 +215,9 @@ export const ELEMENT_TYPE_DC = 'DISTANT CLOUD';
 export const ELEMENT_TYPE_CN = 'CORE NETWORK';
 export const ELEMENT_TYPE_EDGE = 'EDGE';
 export const ELEMENT_TYPE_FOG = 'FOG';
-export const ELEMENT_TYPE_UE = 'UE';
+export const ELEMENT_TYPE_UE = 'TERMINAL';
```
Comment (dilallkx) [inline js-apps/meep-frontend/src/js/meep-constants.js:218]: ELEMENT_TYPE_UE seems to be used while parsing a scenario in scenario-utils.js. This change may break that code. Have you validated that UEs appear correctly in the frontend map?

**Review by dilallkx — CHANGES_REQUESTED** [review]

**Comment by dilallkx** [comment]
Also, please rebaseline to tip of develop branch.

File: js-apps/meep-frontend/src/js/meep-constants.js (hunk 1 above)
Comment (nikhildoifode) [reply js-apps/meep-frontend/src/js/meep-constants.js:233]: Fixed the marker change issue

**Comment by nikhildoifode** [comment]
Rebaselined to the tip of develop branch.

**Review by dilallkx — APPROVED** [review]

**Pull request merged**